pkg/cri/server/sandbox_run_windows.go

```diff
@@ -54,6 +54,30 @@ func (c *criService) sandboxContainerSpec(id string, config *runtime.PodSandboxC
 
 	specOpts = append(specOpts, customopts.WithWindowsDefaultSandboxShares)
 
+	// Start with the image config user and override below if RunAsUsername is not "".
+	username := imageConfig.User
+
+	windowsConfig := config.GetWindows()
+	if windowsConfig != nil {
+		securityCtx := windowsConfig.GetSecurityContext()
+		if securityCtx != nil {
+			runAsUser := securityCtx.GetRunAsUsername()
```

Comment:
`GetSecurityContext` and `GetRunAsUsername` both correctly handle nil receivers. So you can probably just write:

```golang
runAsUser := config.GetWindows().GetSecurityContext().GetRunAsUsername()
if runAsUser != "" {
    username = runAsUser
}
```

Reply:
I guess you also use `securityCtx` below, but still can probably simplify this a little.

Reply:
Sure!

Reply:
Just did the chained calls twice, for the username and then for the cred spec. Seems a bit cleaner, ptal